types/interfaces.ts

```diff
@@ -242,3 +242,24 @@ export enum LogType {
   Request = 'request',
   Misc = 'misc',
 }
+
+export type SchemaQuestion = {
+  reference: string
+  title: string
+  description: string
+  type: string
+  format?: string
+  minLength?: number
+  maxLength?: number
+  widget?: string
+  readOnly?: boolean
+}
+
+export type SchemaPage = {
+  title: string
+  description: string
+  type: string
+  properties: SchemaQuestion
```

Comment:
This doesn't look quite right to me. I thought `properties` was use to store multiple questions. Can't remember if it was a map of key/value pairs or an array, but as this is typed you can only have one `SchemaQuestion` per page.

Reply:
Deleting as it looks like I'm not using it anyway, which is good because typing the format that the library needs is a bit of a pain!